website/docs/getting-started/setup/installation-guides/kubernetes/migrate-to-ee-chart.md

```diff
@@ -83,48 +83,47 @@ Similarly, take a backup of Keycloak:
    07:25:14,488 INFO  [org.keycloak.services] (ServerService Thread Pool -- 57) KC-SERVICES0035: Export finished successfully
    ```
 
-   Press <kbd>ctrl+c</kbd> to stop the process after seeing the above lines.
+   When you see lines similar to this, press <kbd>ctrl+c</kbd> to stop the process.
 
-6. Exit out of the shell with:
+6. To exit out of the shell, run following command:
 
    ```bash
    exit
    ```
 
-7. Download the backup file, available at `/tmp/keycloak_backup.json`:
+7. To download the backup file available at `/tmp/keycloak_backup.json`, run the following command:
 
    ```bash
    kubectl cp <namespace>/appsmith-0:/tmp/keycloak_backup.json ./keycloak_bkp.json
    ```
 
-## Uninstall existing Helm chart
+## Uninstall Helm chart
 
-Let's uninstall the current Appsmith Helm chart.
+To uninstall the existing helm chart, run the following command:
 
 ```bash
 helm uninstall appsmith -n <namespace>
 ```
 
-## Deploy new helm chart
+## Configure parameters
 
-Before we can deploy using the Helm chart from `helm-ee.appsmith.com`, we need to make the following few additions to the configuration in `values.yaml`.
+To ensure that the Business Edition Helm chart runs, you need to make some changes to the `values.yaml` file. Follow the steps below to configure parameters:
 
-1. Enabling PostgreSQL: Add the following snippet to the bottom of your `values.yaml` file.
+1. Add the following snippet to the bottom of your `values.yaml` file. This snippet enables PostgreSQL.
 
    ```yaml
-
    postgresql:
      enabled: true
      auth:
        username: root
-       password: password
-       postgresPassword: password
+       password: <PASSWORD>
+       postgresPassword: <POSTGRESQL_PASSWORD>
        database: keycloak
    ```
 
-2. [Choose, or create a shared file system](./business-edition#create-a-shared-file-system)
+2. Choose, or create a shared file system. For more information, see [Create a shared file system.](/getting-started/setup/installation-guides/kubernetes/business-edition#create-a-shared-file-system)
```

Comment:
I'd request undoing this change as well. The given link is not _more_ information, it's _the_ information. It's not optional, like, _see this link if you want more information_. It's more like, _follow instructions from this link_, if I'm making sense. 😅

Reply:
I thought users would get more information on this. So added it like that.
In that case, I think it will be back and forth for users to navigate to another page, and then perform the action and move back to this page to proceed.  I guess embedding that info here would make more sense. CC: @danciaclara - Your thoughts?

Reply:
As discussed, embed the section 'create a shared file system' here.